src/object/obj_tx.c

```diff
@@ -394,7 +394,7 @@ dc_tx_cleanup_one(struct dc_tx *tx, struct daos_cpd_sub_req *dcsr)
 
 		if (dcsr->dcsr_sgls != NULL) {
 			for (i = 0; i < dcsr->dcsr_nr; i++)
-				daos_sgl_fini(&dcsr->dcsr_sgls[i],
+				d_sgl_fini(&dcsr->dcsr_sgls[i],
```

Comment:
Alignment, but I wouldn't refresh for it.

Reply:
done